airbyte-ci/connectors/connector_ops/connector_ops/utils.py

```diff
@@ -119,12 +120,14 @@ def get_changed_file(file_name: str, diff_regex: Optional[str] = None) -> Set[st
     else:
         diff_command_args = ("--name-only", f"-G{diff_regex}", DIFFED_BRANCH)
 
-    changed_acceptance_test_config_paths = {
+    modified_files = airbyte_repo.git.diff(*diff_command_args).split("\n")
+    changed_connector_paths = {
         file_path
-        for file_path in airbyte_repo.git.diff(*diff_command_args).split("\n")
-        if file_path.startswith(SOURCE_CONNECTOR_PATH_PREFIX) and file_path.endswith(file_name)
+        for file_path in modified_files
+        if (file_path.startswith(SOURCE_CONNECTOR_PATH_PREFIX) or file_path.startswith(DESTINATION_CONNECTOR_PATH_PREFIX))
+        and file_path.endswith(file_name)
```

Comment:
Note that if we want this to apply to destinations we also have to modify the action itself, which currently has a path filter for sources